tests/functionality_tests/sensor_physical_testing.py

```diff
@@ -0,0 +1,42 @@
+from engine.robot_state import Robot_State
+from engine.robot import Robot
+from electrical.gps import GPS
+from engine.grid import Grid
+from engine. database import DataBase
+import math
+
+if __name__ == "__main__":
+    # make sure to go to imu and gps and set import to true
+    test_state = 0  # 0 is no node straight, 1 is node straight, 2 is turn, -1 is sensor
+    rb_state = Robot_State(0, 0, 0, .2, .5, .2)
+    rb = Robot(rb_state)
```

Comment:
again, try to be consistent with naming

Reply:
em, what naming is bad?

Reply:
rb -> robot or r2d2

Reply:
rb is just an example of a robot. isn't r2d2 just an example of a robot? I can call it r2d2 but I'm just calling it rb. Should I just call it r2d2 then?